test/integration/study_validation_test.rb

```diff
@@ -0,0 +1,207 @@
+require "integration_test_helper"
+
+class StudyAdminTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+
+  setup do
+    @test_user = User.find_by(email: 'testing.user@gmail.com')
+    @sharing_user = User.find_by(email: 'sharing.user@gmail.com')
+    auth_as_user(@test_user)
+    sign_in @test_user
+    @random_seed = File.open(Rails.root.join('.random_seed')).read.strip
+  end
+
+  # check that file header/format checks still function properly
+  test 'should fail all parse jobs' do
+    puts "#{File.basename(__FILE__)}: #{self.method_name}"
+    study_params = {
+        study: {
+            name: "Parse Failure Study #{@random_seed}",
+            user_id: @test_user.id
+        }
+    }
+    post studies_path, params: study_params
+    follow_redirect!
+    assert_response 200, "Did not redirect to upload successfully"
+    @study = Study.find_by(name: "Parse Failure Study #{@random_seed}")
+    assert @study.present?, "Study did not successfully save"
```

Comment:
I agree - it seems like a lot of repetition.  I'll add the line breaks, but keep the idea of extracting common calls out to separate functions in the back of my mind in case we get back to updating this.